polkadot/primitives/src/vstaging/mod.rs

```diff
@@ -447,7 +457,7 @@ pub fn skip_ump_signals<'a>(
 impl CandidateCommitments {
 	/// Returns the core selector and claim queue offset determined by `UMPSignal::SelectCore`
 	/// commitment, if present.
-	pub fn core_selector(
+	fn core_selector_before_approved_peer_allowed(
```

Comment:
I don't really see why we want to change the functionality of this method and it's name. It can just return the core selector as it used to. If there are other signals, so be it. It should ignore them.

Reply:
done